webapp/maps_heatmaps/.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
+  "projects": {
+    "default": "step-project-ellispis"
+  }
+}
```

Comment:
Can you add a docstring at the top of the file with a URL pointer on why we need this file and what it does?

Reply:
added.